locallibrary/catalog/models.py

```diff
@@ -1,3 +1,84 @@
 from django.db import models
 
-# Create your models here.
+# Used to generate URLs by reversing the URL patterns
+from django.urls import reverse
+import uuid  # Required for unique book instances
+
+
+class Genre(models.Model):
+    """Model representing a book genre."""
+    name = models.CharField(
+        max_length=200, help_text='Enter a book genre (e.g.Science Fiction)')
```

Comment:
Dùng django I18n để translate text của option `help_text` em nhé, mục đích cho tính năng đa ngôn ngữ
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#internationalization-in-python-code